torchqdynamics/solvers/_base.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+import torch
+
+
+@dataclass
+class BaseSolverOption:
+    H: torch.Tensor
+    jump_ops: List[torch.Tensor]
+    expect_ops: List[torch.Tensor]
+    t_save: np.array
```

Comment:
If `H`, `expect_ops`, `t_save` and `store_states` are common to everyone, shouldn't they be part of the `mesolve` arguments then? I put `H` in the dataclass for now but afterwards I'm realizing it may not be the best choice

Reply:
I agree 👍 So we restrict the use of dataclasses to the solvers options.